backend/scraper/scrape_walmart.py

```diff
@@ -1 +1,88 @@
-# TODO scrape items from Walmart
+from selenium import webdriver
+from bs4 import BeautifulSoup
+import requests
+
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.keys import Keys
+import time
+
+ZIPCODE = '77841'
+
+driver_path = r'C:/Users/Ryan/Desktop/Web Drivers/chromedriver.exe'

```

Comment:
There's probably a better way to handle this than hard-coding the path. Usually we use environment variables or config files